packages/@aws-cdk/cfnspec/build-tools/regen-cfnonly-readmes.ts

```diff
@@ -0,0 +1,31 @@
+// Helper script to regen all CFN-only library readmes from the template.
```

Comment:
How about just run this every time during cfnspec bump?

Otherwise, no one is going to know about this besides you.

Reply:
I think this comment is trying to make sure new modules are being created with the right READMEs.

That already happens, the READMEs are passing through the same code path.

Reply:
Isn't this script also meant to be run when the "template" changes?  Re-gen on all L1 modules?

Running this each time would mean that any missed corrections will be caught (at least during bump).